client/my-sites/checkout/controller.jsx

```diff
@@ -48,8 +47,7 @@ const debug = debugFactory( 'calypso:checkout-controller' );
 export function checkout( context, next ) {
 	const { feature, plan, purchaseId } = context.params;
 
-	const user = userFactory();
-	const isLoggedOut = ! user.get();
+	const isLoggedOut = ! isUserLoggedIn( context.store.getState() );
 	const state = context.store.getState();
 	const selectedSite = getSelectedSite( state );
 	const currentUser = getCurrentUser( state );
```

Comment:
Here we could use `getCurrentUserVisibleSiteCount`

Reply:
Done as part of c2bb0c5f7413926db2a9c6121e8a617a37fa3a0b

Reply:
Thanks! I think this can be merged now :+1: